split.Console.Tests/Models/Blackjack/BlackjackPlayerTests.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Xunit;
+using split.Console.Models;
+using split.Console.Models.Blackjack;
+
+namespace split.Console.Tests.Models.Blackjack
+{
+    public class BlackjackPlayerTests
+    {
+        // A player should be able to be dealt a card
+        [Fact]
+        public void HasOneCard()
+        {
+            Card card = new Card(SuitType.Club, RankType.Ace);
+            BlackjackPlayer player = new BlackjackPlayer("Test", "");
+            player.AddCardToHand(card);
+
+            Assert.Equal(1, player.NumberCardsInHand());
+        }
+
+        // A player should be busted if they have a sum of 23
+        [Fact]
+        public void PlayerBusts()
+        {
```

Comment:
I don't think this test will pass because the Player object is not available in this test's scope.